cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/AbstractStreamCoordinatorClient.java

```diff
@@ -53,30 +39,23 @@
 public abstract class AbstractStreamCoordinatorClient extends AbstractIdleService implements StreamCoordinatorClient {
 
   private static final Logger LOG = LoggerFactory.getLogger(AbstractStreamCoordinatorClient.class);
-  private static final Gson GSON = new GsonBuilder()
-    .registerTypeAdapter(Schema.class, new SchemaTypeAdapter())
-    .create();
+  private static final Gson GSON = new Gson();
```

Comment:
If we're using this GSON instance to serialize/deserialize ``CoordinatorStreamProperties``, which contain a ``Schema`` - via the ``Formatspecification`` class, shouldn't we leave the ``SchemaTypeAdapter`` here?